### ISSUE TITLE
BUG: Virtual keyboard disappearing in Windows tablet mode

### ISSUE THREAD

**Issue by teacupx** [sumatrapdfreader/sumatrapdf#1901]
Hello. I am using SumatraPDF v32 64-bit, in a 2-in-1 convertible laptop with touchscreen. 

Every time I am using the device on tablet mode (therefore, physical keyboard automatically disabled), and I need to type something (e.g. in the search box), the virtual keyboard disappears instantly after appearing.

The problem is that, when the virt keyboard appears, Sumatra resizes the window to fit the remaining space of the screen. But instantly, the window is automatically resized back to full screen, making the keyboard disappear.

The problem is only present when using tablet mode. In Windows "normal" mode, you can use the virtual keyboard without problems.

**Comment by GitHubRulesOK** [comment]
I see the same rescaling problem when using Acrobat to search via virtual keyboard entry thus think it is more a windows generic pop-up bug rather than specific to SumatraPDF.
I have not tried other workarounds but my addin to search using console and VBS is less affected since it is outside the keyboard zone of influence. Similar if I switch off toolbar and use the smaller CTRL+F dialog it is only that that is moved above keboard
![image](https://user-images.githubusercontent.com/12754913/111184051-7c385a80-85a8-11eb-8bb9-67e27ed4a708.png)


@kjk I suggest the "easy" solution to this issue is to make the **search dialog box** ALWAYS floating

**Comment by teacupx** [comment]
You're right, I have the same problem with PDF Xchange too. But other programs do not have the problem.

The workaround you mention is good for the search bar, but still when you need to type a page number in the number box, you will encounter the same problem.

My workaround is simply to disable tablet mode when I need to type something.

**Comment by GitHubRulesOK** [comment]
My addin at present is designed to only search forward from current page # but could be adapted to ask for a page # to start from however for your specific needs I would be rewriting a floating launch tool to do those and more the problem is finding one that's easy to configure to work with SumatraPDF without too much re-inventing. At present am testing out this one
![image](https://user-images.githubusercontent.com/12754913/111195314-0afea480-85b4-11eb-9eb1-9d373f543e2c.png)

**Comment by kjk** [comment]
Closing: not enough information and no activity since 2021. This looks like a Windows tablet-mode / touch-keyboard shell interaction rather than a clear Sumatra bug, and we don't have a current repro on modern Windows builds.

If this still happens on a current pre-release, reopen with Windows version, Sumatra version, and exact steps (which field: search, find, password, etc.).
